pkgs/development/interpreters/erlang/R23.nix

```diff
@@ -4,5 +4,5 @@
 # nix-prefetch-url --unpack https://github.com/erlang/otp/archive/OTP-${version}.tar.gz
 mkDerivation {
   version = "23.3.1";
-  sha256 = "1nx9yv3l8hf37js7pqs536ywy786mxhkqba1jsmy1b3yc6xki1mq";
+  sha256 = "sha256-uIY4u2F+rOCrlkEtPGGvBh3PvRlF43u0PMNBRMf2qds=";
```

Comment:
Why did this hash change?

Reply:
As a result of running the script, I guess it's the same hash but in the new format. I'll change it back!

Reply:
The script shouldn't change the hash if the version hasn't increased, actually, what happen is that I replaced this file with an older version to check that it was able to update it, and it ended up with the new hash format.